python/vowpalwabbit/pyvw.py

```diff
@@ -338,25 +328,40 @@ def get_prediction(ec, prediction_type: Union[int, PredictionType]):
     out : integer/list
         Prediction according to parameter prediction_type
     """
-    if isinstance(prediction_type, PredictionType):
-        prediction_type = prediction_type.value
-
-    switch_prediction_type = {
-        pylibvw.vw.pSCALAR: ec.get_simplelabel_prediction,
-        pylibvw.vw.pSCALARS: ec.get_scalars,
-        pylibvw.vw.pACTION_SCORES: ec.get_action_scores,
-        pylibvw.vw.pACTION_PROBS: ec.get_action_scores,
-        pylibvw.vw.pMULTICLASS: ec.get_multiclass_prediction,
-        pylibvw.vw.pMULTILABELS: ec.get_multilabel_predictions,
-        pylibvw.vw.pPROB: ec.get_prob,
-        pylibvw.vw.pMULTICLASSPROBS: ec.get_scalars,
-        pylibvw.vw.pDECISION_SCORES: ec.get_decision_scores,
-        pylibvw.vw.pACTION_PDF_VALUE: ec.get_action_pdf_value,
-        pylibvw.vw.pPDF: ec.get_pdf,
-        pylibvw.vw.pACTIVE_MULTICLASS: ec.get_active_multiclass,
-        pylibvw.vw.pNOPRED: ec.get_nopred,
+    warnings.warn(
+        "get_prediction is deprecated, use Example.get_prediction instead.",
+        DeprecationWarning,
+    )
+    return ec.get_prediction(prediction_type)
+
+
+def get_label_class_from_enum(
+    label_type: LabelType,
+) -> Union[
+    "SimpleLabel",
+    "MulticlassLabel",
+    "CostSensitiveLabel",
+    "CBLabel",
+    "CCBLabel",
+    "SlatesLabel",
+    "CBContinuousLabel",
+]:
```

Comment:
Can we use `AbstracLabel` here instead?

Reply:
Ah I just realized this is wrong anyway. It should be Type[AbstractLabel] but I don't think we can do that until 3.9. I'll leave this one unannotated

Reply:
Actually it is available